### DISCUSSION TITLE
flutter_modular na Web: como retirar o "#" do endereço?

### DISCUSSION THREAD

**Discussion by nilopaim** [Flutterando/forum#11] [Perguntas e Respostas] [answered]
Usando o package flutter_modular aqui mas gerando Web. 

O endereço aparece na barra do browser como `xyz.com.br/#/pageName`

Como retirar esse "#"?

**Comment by GabrielCR99** [comment, marked as answer]
Vc pode usar um package pra poder remover o # na barra do browser, como esse: https://pub.dev/packages/url_strategy

**Comment by nilopaim** [comment]
Perfeito, Gabriel!

Grato!

**Comment by jacobaraujo7** [comment]
Caramba essa eu não sabia!

**Comment by nilopaim** [comment]
Achei isso também, que além de retirar a "#" ainda desabilita o botão "back" do browser. Ou seja, quem faz a navegação entre páginas é você mesmo.

In order to disable all browser history, you could do something like this:

```import 'package:flutter_web_plugins/url_strategy.dart';

void main() {
  // Before running your app:
  setUrlStrategy(null);

  runApp(...);
}```

Of course, you'll need to add the flutter_web_plugins package to your pubspec.yaml:

```dependencies:
  flutter_web_plugins:
    sdk: flutter
```

Fonte: [https://github.com/flutter/flutter/issues/108114#issuecomment-1234657755](url)

**Reply by GabrielCR99** [reply]
Mano, essa aí do desabilitar o "back" do browser eu não sabia que existia! E olha que isso aí era muito chato pra tentar implementar. Essa dica salvou em :D
